src/main/resources/templates/reservation_search.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+  <p> Search for Available rooms</p>
+
+  <form action="/hotel/search" method="post">
+    <label for="Date" required>Date:</label>
+    <input type="date" id="Date" name="Date" value=""><br>
```

Comment:
Default value to explain expected input would be a good add.

Reply:
backlog